stories/CommentSummary.stories.tsx

```diff
@@ -0,0 +1,130 @@
+/* eslint-disable react/prop-types */
+import * as React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import CommentSummary from '../src/components/CommentSummary';
+import { createInternalComment } from '../src/reducers/comments';
+import { createFakeExternalComment } from '../src/test-helpers';
+import { loremIpsum } from './utils';
+
+const render = ({ comments = [createFakeExternalComment()] } = {}) => {
+  return (
+    <CommentSummary comments={comments.map((c) => createInternalComment(c))} />
+  );
+};
+
+export const createOneLineComment = () => [
```

Comment:
I made this function and the one below because the next story for `CommentSummaryButton` is going to use them.